crates/ruma-client-api/src/future/future_message_event.rs

```diff
@@ -0,0 +1,181 @@
+//! `PUT /_matrix/client/*/rooms/{roomId}/send_future/{eventType}/{txnId}`
+//!
+//! Send a future (a scheduled message) to a room. [MSC4140](https://github.com/matrix-org/matrix-spec-proposals/pull/4140)
+
+pub mod v3 {
```

Comment:
The module name should be `unstable` and use the unstable prefixes defined in the MSC.

This comment is valid for the other endpoints as well.